lib/lettering/font.py

```diff
@@ -365,7 +365,7 @@ def _add_trims(self, destination_group, text, trim_option, use_trim_symbols, bac
         line_break_indices = [i for i, t in enumerate(text) if t == "\n"]
         for group in destination_group.iterdescendants(SVG_GROUP_TAG):
             # make sure we are only looking at glyph groups
-            if group.get("id") != "glyph":
+            if not group.get("id", "").startswith("glyph"):
```

Comment:
Yup, this is the best we can do.  I am kicking myself because using the ID to identify stuff we created is not a great idea, but it's too late to change :(